sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -853,6 +855,82 @@ def peek_messages(
         with receive_trace_context_manager(self, span_name=SPAN_NAME_PEEK, links=links, start_time=start_time):
             return messages
 
+    def delete_messages(
+        self,
+        *,
+        max_message_count: Optional[int] = None,
+        before_enqueued_time_utc: Optional[datetime.datetime] = None,
+    ) -> int:
+        """
+        This operation deletes messages in the queue that are older than the specified enqueued time.
+
+        :keyword int or None max_message_count: The maximum number of messages to delete.
```

Comment:
Should we specify what None will do here without mentioning a number explicitly? Like, `If None, the maximum number will be determined by the service.` Or would not mentioning at all be best?